openeuler-github-actions-ascend310--22.03.sp2/Dockerfile

```diff
@@ -22,22 +22,25 @@ RUN yum install -y \
     gtk3-devel
 
 # Install CANN
-RUN wget https://ascend-repo.obs.cn-east-2.myhuaweicloud.com/CANN/CANN%207.0.RC1/Ascend-cann-toolkit_7.0.RC1_linux-"$(uname -i)".run && \
-    chmod +x Ascend-cann-toolkit_7.0.RC1_linux-"$(uname -i)".run && \
-    ./Ascend-cann-toolkit_7.0.RC1_linux-"$(uname -i)".run --quiet --install && \
-    rm -f ./Ascend-cann-toolkit_7.0.RC1_linux-"$(uname -i)".run
+RUN wget https://ascend-repo.obs.cn-east-2.myhuaweicloud.com/Milan-ASL/Milan-ASL%20V100R001C15SPC703/Ascend-cann-toolkit_7.0.0.alpha003_linux-"$(uname -i)".run && \
+    chmod +x Ascend-cann-toolkit_7.0.0.alpha003_linux-"$(uname -i)".run && \
+    ./Ascend-cann-toolkit_7.0.0.alpha003_linux-"$(uname -i)".run --quiet --install && \
+    rm -f ./Ascend-cann-toolkit_7.0.0.alpha003_linux-"$(uname -i)".run
 
 # Install kernels
-RUN wget https://ascend-repo.obs.cn-east-2.myhuaweicloud.com/CANN/CANN%207.0.RC1/Ascend-cann-kernels-310p_7.0.RC1_linux.run && \
-    chmod +x Ascend-cann-kernels-310p_7.0.RC1_linux.run && \
-    ./Ascend-cann-kernels-310p_7.0.RC1_linux.run --quiet --install && \
-    rm -f ./Ascend-cann-kernels-310p_7.0.RC1_linux.run
+RUN wget https://ascend-repo.obs.cn-east-2.myhuaweicloud.com/Milan-ASL/Milan-ASL%20V100R001C15SPC703/Ascend-cann-kernels-310p_7.0.0.alpha003_linux.run && \
+    chmod +x Ascend-cann-kernels-310p_7.0.0.alpha003_linux.run && \
+    ./Ascend-cann-kernels-310p_7.0.0.alpha003_linux.run --quiet --install && \
+    rm -f ./Ascend-cann-kernels-310p_7.0.0.alpha003_linux.run
 
 # Self-hosted runner UID is 1001, matches user id on host.
 RUN groupadd -g 1000 HwHiAiUser && \
     useradd ci -m -s /bin/bash -G users --uid=1001 && \
     usermod -a -G HwHiAiUser ci
 
+# Workaround for AscendC build bug with ninja, remove this line for next cann version.
+RUN sed -i "/\$(MAKE)/d" /usr/local/Ascend/ascend-toolkit/latest/compiler/tikcpp/ascendc_kernel_cmake/function.cmake
```

Comment:
A month was past. Is there a newer version with this issue addressed?

Reply:
Yes, there's a new version now, I will update this dockerfile.

Reply:
Tests passed, and the latest version (8.0.RC1.alpha001) of CANN can solve this issue.